frontend/src/app/core/services/notification.service.ts

```diff
@@ -9,6 +10,8 @@ import { INotificationModel } from '@shared/models/notifications/notifications';
 export class NotificationService {
     private notifications: INotificationModel[] = [];
 
+    private strongNotifications: TypeNotification[] = [TypeNotification.FriendRequest];
```

Comment:
what is this? not clear name

Reply:
Strong means that notification won't disappear when we close notifications modal window. User will have to get rid of it manually by accepting or rejecting friend request

Reply:
I believe we don't need this